lib/EscriptGrammar/EscriptLexer.g4

```diff
@@ -1,6 +1,12 @@
 lexer grammar EscriptLexer;
 channels { COMMENTS }
 
+@lexer::members
```

Comment:
The lexer updates are primarily based off of C# here: https://github.com/antlr/grammars-v4/blob/master/csharp/CSharpLexer.g4 , minus the use of "verbatim string"